templates/test/comments.html.ep

```diff
@@ -0,0 +1,37 @@
+% if (!$job->comments) {
```

Comment:
I was trying to track this down, but I'm missing something. You use $job->comments as an ARRAY everywhere, but isn't $job->comments a resultset? Obviously it seems to work, except that I found no 'No comments' on gertrude.